Makefile

```diff
@@ -238,6 +238,17 @@ pylint_rc_file := pylintrc
 pylint_todo_opts := --disable=fixme
 pylint_no_todo_opts := --enable=fixme
 
+# Disable the similarity checker on pylint <2.10 due to issues #2672, #2673
```

Comment:
getting picky, Why not indent like much of Makefile

Reply:
Agreed. Will indent